test/optimizations/sungeun/optimized-on/test_OptimizedOnAtomic.chpl

```diff
@@ -0,0 +1,24 @@
+
+on Locales[numLocales-1] {
+  var y:atomic int;
+
+  on Locales[0] {
+
+    on Locales[numLocales-1] {
+      local atomic_fetch_add_explicit_int_least64_t(y._v, 1, memory_order_seq_cst);
```

Comment:
Yes, I removed the local block and the test still works. I wanted to test that the 'on' statement inside of Atomics.chpl's implementations of things like atomic int.sub is a fast execute-on. This test reproduces that code.

If this test called e.g. atomic int.fetchAdd, that call would contain an on statement inside of it, and the conversion to a fast executeOn would not fire (unless the whole business was in a local block).